common/src/main/java/org/apache/cassandra/sidecar/common/data/SSTableComponent.java

```diff
@@ -35,8 +39,21 @@ public class SSTableComponent
      * @param componentName      the name of the SSTable component
      */
     public SSTableComponent(QualifiedTableName qualifiedTableName, String componentName)
+    {
+        this(qualifiedTableName, null, componentName);
+    }
+
+    /**
+     * Constructor for the holder class
+     *
+     * @param qualifiedTableName the qualified table name in Cassandra
+     * @param indexName          the name of the index for the SSTable component
+     * @param componentName      the name of the SSTable component
+     */
+    public SSTableComponent(QualifiedTableName qualifiedTableName, @Nullable String indexName, String componentName)
```

Comment:
Is it possible to just modify the existing constructor? Let's try to not add more code.